[CICD] ECS에서 환경변수 못 읽는 문제 발생

## 개요
- ECS에서 서비스 배포가 제대로 안되고 롤백됨

## 상세 설명
- 에러로그
```
Caused by: org.springframework.util.PlaceholderResolutionException:  
Could not resolve placeholder 'JWT_ACCESS_TOKEN_SECRET' 
in value "${JWT_ACCESS_TOKEN_SECRET}" <-- "${playlist.jwt.access-token.secret}"
```

--- a/.github/workflows/deploy.yml
+++ b/.github/workflows/deploy.yml
@@ -45,7 +45,7 @@ jobs:
              '.executionRoleArn |= sub("ACCOUNT_ID_PLACEHOLDER"; $ACCOUNT_ID)
               | .taskRoleArn |= sub("ACCOUNT_ID_PLACEHOLDER"; $ACCOUNT_ID)' \
              task-definition.json > tmp.json && mv tmp.json task-definition.json
-
+          
       - name: Update image and env vars using jq # 이미지 + 환경변수 치환
         run: |
           jq \

